gcloud/storage/bucket.py

```diff
@@ -205,6 +205,11 @@ def path(self):
 
         return self.path_helper(self.name)
 
+    @property
```

Comment:
I'm not sure what you mean by before? You want me to re-write all the commits or move it up vertically in a file? Also, all the PRs strung together bring us to a "final state" and most states in between are just transient.